src/helper.rs

```diff
@@ -107,6 +109,61 @@ mod html {
         fn render(&self) -> String;
     }
 
+    pub fn superscript(n: u8) -> String {
+        if n == 0 {
+            "⁰".to_owned()
+        } else if n == 1 {
+            "¹".to_owned()
+        } else if n == 2 {
+            "²".to_owned()
+        } else if n == 3 {
+            "³".to_owned()
+        } else if n == 4 {
+            "⁴".to_owned()
+        } else if n == 5 {
+            "⁵".to_owned()
+        } else if n == 6 {
+            "⁶".to_owned()
+        } else if n == 7 {
+            "⁷".to_owned()
+        } else if n == 8 {
+            "⁸".to_owned()
+        } else if n == 9 {
+            "⁹".to_owned()
+        } else if n >= 10 {
+            superscript(n / 10) + &superscript(n % 10)
+        } else {
+            n.to_string()
+        }
+    }
```

Comment:
We can write like:

```rust
pub fn superscript<'a>(n: u8) -> &'a [u8] {
    match u {
        0 => b"⁰",
        // ...
    }
}
```
And this will be lower costs : )